backend/database.js

```diff
@@ -148,34 +162,41 @@ export default (sbp('sbp/selectors/register', {
   //
   // TODO: add encryption
   // =======================
-  'backend/db/readFile': async function (filename: string): Promise<*> {
+  'backend/db/readFile': async function (filename: string): Promise<Buffer | Error> {
     const filepath = throwIfFileOutsideDataDir(filename)
     if (!fs.existsSync(filepath)) {
       return Boom.notFound()
     }
-    return await readFileAsync(filepath)
+    return await readFile(filepath)
   },
-  'backend/db/writeFile': async function (filename: string, data: any): Promise<*> {
+  'backend/db/writeFile': async function (filename: string, data: any): Promise<void> {
     // TODO: check for how much space we have, and have a server setting
     //       that determines how much of the disk space we're allowed to
     //       use. If the size of the file would cause us to exceed this
     //       amount, throw an exception
-    return await writeFileAsync(throwIfFileOutsideDataDir(filename), data)
+    return await writeFile(throwIfFileOutsideDataDir(filename), data)
   },
-  'backend/db/writeFileOnce': async function (filename: string, data: any): Promise<*> {
+  'backend/db/writeFileOnce': async function (filename: string, data: any): Promise<void> {
```

Comment:
Could you get rid of this and consolidate the `/file` API in `routes.js` to use `'chelonia/db/get` and `'chelonia/db/set'` instead?

Reply:
`'chelonia/db/set'` currently only support strings. Should it be extended for blob support?

Reply:
Let's figure this out on Slack, I'll reply there